README.md

```diff
@@ -1,8 +1,75 @@
-# Laminar
+# Laminar | Game Networking Protocol
 
-[![Build Status](https://travis-ci.org/amethyst/laminar.svg?branch=master)](https://travis-ci.org/amethyst/laminar)
+[![Latest Version](https://img.shields.io/crates/v/laminar.svg)](https://crates.io/crates/laminar) | [![Build Status](https://travis-ci.org/amethyst/laminar.svg?branch=master)](https://travis-ci.org/amethyst/laminar) | [![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](./LICENSE) | [![docs.rs](https://docs.rs/laminar/badge.svg)](https://docs.rs/laminar/) | [Examples](https://github.com/amethyst/laminar/tree/master/examples) | [Changelog](LINK) | [Release Nodes](LINK)
+|----|----|----|----|----|----|----
 
-A UDP protocol that provides partial reliablity. Coming soon!
+A UDP protocol that provides partial reliability. Coming soon!
+
+## Table of contents:
+- [Useful links](https://github.com/amethyst/laminar#useful-links)
+- [Features](https://github.com/amethyst/laminar#features)
+- [Examples](https://github.com/amethyst/laminar#examples)
+    - [Udp](https://github.com/amethyst/laminar#udp)
+- [Notice](https://github.com/amethyst/laminar#notice)
+- [Contributing](https://github.com/amethyst/laminar#contributing)
+- [Authors](https://github.com/amethyst/laminar/#authors)
+- [License](#license)
+
+Add the laminar package to your `Cargo.toml` file.
+
+```
+[dependencies]
+laminar = "0.0.0"
+
+```
+And import the laminar modules you want to use.
+
+```rust
+extern crate laminar;
+
+// this module contains all socket related logic.
+use self::laminar::net::{UdpSocket, SocketAddr, NetworkConfig, Connection, Quality, TcpSocket, TcpClient, TcpSocketState};
+// this module contains packet related logic.
+use self::laminar::packet::{Packet};
+```
+
+## Useful Links
+
+- [Documentation](https://docs.rs/laminar/).
+- [Cargo Page](https://crates.io/crates/laminar)
+- [examples](https://github.com/amethyst/laminar/tree/master/examples)
+
+## Features
+These are the features from this crate:
+
+- Semi-reliable UDP
+- UDP
+- TCP
+
+## Examples
+These are some basic examples demonstrating how to use this crate. See [examples](https://github.com/amethyst/laminar/tree/master/examples) for more.
+
+### Udp API | [see more](LINK)
```

Comment:
Not familiar with this syntax...what is LINK?

Reply:
I believe its just a place holder. We need to actually add the example here.

Reply:
jup, examples was not merged jet but if we have good examples this will be a link to there.

Reply:
Gotcha